tests/opentracer/test_tracer_tornado.py

```diff
@@ -0,0 +1,31 @@
+import pytest
+from nose.tools import eq_, ok_
+
+from opentracing.ext.scope_manager.tornado import TornadoScopeManager
```

Comment:
https://github.com/DataDog/dd-trace-py/blob/opentracer/tox.ini#L90-L93 -> can you update the PR so it uses the version 2.0.0?